pride/src/integTest/data/PrideInitIntegrationTest/git-module/build.gradle

```diff
@@ -17,7 +17,7 @@ configurations {
 	compile
 }
 
-dynamicDependencies {
+dependencies {
```

Comment:
This was intentionally left as `dynamicDependencies` so that there's at least one test that exercises the backwards compatibility support. That said, it should probably be a separate integration test. In any case, please revert it for this pull-request.